File: indiana-backend/product-api/src/main/java/com/indiana/product/api/product/domain/Product.java
package com.indiana.product.api.product.domain;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "products")
public class Product {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  @Column(name = "pro_id")
  private Long id;
  @Column(name = "pro_code")
  private String code;
  @Column(name = "pro_barcode")
  private String barcode;
  @Column(name = "pro_name")
  private String name;
  @Column(name = "pro_description")
  private String description;
  @Column(name = "pro_packed_weight")
  private Double packedWeight;
  @Column(name = "pro_packed_width")
  private Double packedWidth;
  @Column(name = "pro_packed_depth")
  private Double packedDepth;
  @Column(name = "pro_refrigerated")
  private Boolean refrigerated;
}

File: indiana-backend-go/internal/core/domain/product.go
package domain

import (
	"time"
)

type Product struct {
	ID           int64
	Code         string
	BarCode      string
	Name         string
	Description  string
	PackedWeight float64
	PackedWidth  float64
	PackedDepth  float64
	CreatedAt    time.Time
	UpdatedAt    time.Time
}
